data/human/fleets.txt

```diff
@@ -2004,6 +2028,10 @@ fleet "Small Independent"
 		"Clipper (Heavy)"
 	variant 1
 		"Clipper (Speedy)"
+	variant 6
+		"Scrapper"
+	variant 4
+		"Scrapper (Minimal)"
```

Comment:
these feel too likely to be honest
how about you cut the weight in two and multiply the amount of them by 2?